lib/pgpm/package/make.rb

```diff
@@ -3,21 +3,32 @@
 module Pgpm
   class Package
     module Make
-      def build_steps
-        return [Pgpm::Commands::Make.new("PG_CONFIG=$PG_CONFIG")] if makefile_present?
 
+      def build_steps
+        case Pgpm::OS.in_scope.class.name
+        when "debian", "ubuntu"
+          return []
+        when "rocky+epel-9", "redhat", "fedora"
+          return [Pgpm::Commands::Make.new("PG_CONFIG=$PG_CONFIG")] if makefile_present?
```

Comment:
And speaking of special-casing and the case I expanded on when it comes to configuring – this is the same issue – and an opportunity – to unify the procedure.